py.Dockerfile

```diff
@@ -0,0 +1,30 @@
+# A Dockerfile that sets up a full Gym install with test dependencies
+ARG PYTHON_VER
+FROM python:$PYTHON_VER
+RUN apt-get -y update && apt-get install -y unzip libglu1-mesa-dev libgl1-mesa-dev libosmesa6-dev xvfb patchelf ffmpeg
+RUN \ 
+# Download mujoco
+    mkdir /root/.mujoco && \
+    cd /root/.mujoco  && \
+    curl -O https://www.roboti.us/download/mjpro150_linux.zip  && \
+    unzip mjpro150_linux.zip && \
+    curl -O https://www.roboti.us/download/mujoco200_linux.zip && \
+    unzip mujoco200_linux.zip && \
+    mv mujoco200_linux mujoco200
+
+ARG MUJOCO_KEY
+ARG PYTHON_VER
```

Comment:
This seems to be here twice.

Reply:
apparently, FROM overwrites build arguments, so it does have to be here twice... I know, that is a pretty strange behavior :/